src/openforms/formio/components/np_family_members/tests/test_family_members.py

```diff
@@ -24,8 +25,17 @@
 from ..models import FamilyMembersTypeConfig
 from ..stuf_bg import get_np_children_stuf_bg
 
+TEST_FILES = Path(__file__).parent.resolve() / "responses"
+
 
 class FamilyMembersCustomFieldTypeTest(TestCase):
+    def setUp(self):
+        super().setUp()
+
+        # ensure the schema cache is cleared before and after each test
```

Comment:
I'm not too convinced about it, mixins tend to be an antipattern and hard to keep track of, while setting up the cleanup explicitly makes it much more visible. I particularly don't like generic mixins like `ClearCacheMixin` that clear just about anything because they don't know what's used, and that also makes tests slower.

DRY in test code is something to not chase tbh.